src/app/core/services/githubevent.service.ts

```diff
@@ -32,25 +32,27 @@ export class GithubEventService {
   }
 
   /**
-   * Returns the latest github event (e.g renaming an issue's title) of current repository
-   * @returns the json data of the latest event
+   * Returns the result whether the latest github event (e.g renaming an issue's title)
+   * of current repository has been retrieved or not.
+   * @returns true if a new github event has been retrieved.
```

Comment:
To be more accurate, we are returning `true` if the issues were fetched from GitHub